packages/grafana-runtime/src/utils/returnToPrevious.ts

```diff
@@ -0,0 +1,20 @@
+interface ReturnToPreviousData {
+  title: string;
+  href: string;
+}
+
+type ReturnToPreviousHook = () => (rtp: ReturnToPreviousData) => void;
+
+let rtpHook: ReturnToPreviousHook | undefined = undefined;
+
+export const setReturnToPreviousHook = (hook: ReturnToPreviousHook) => {
+  rtpHook = hook;
+};
+
+export const useReturnToPrevious: ReturnToPreviousHook = () => {
+  if (!rtpHook) {
+    throw new Error('rtpHook not defined yet');
```

Comment:
Maybe, instead of hard erroring, lets just log a warning to the console and not break the app for users

```suggestion
    if (process.env.NODE_ENV !== 'production') {
      throw new Error("useReturnToPrevious hook not found in @grafana/runtime")
    }
    return () => console.error("ReturnToPrevious hook not found")
```